Automattic/simplenote-macos
# Shortcut: Append to note

### Fix
One of the most likely and useful functions we could add to Simplenote shortcuts is the ability to append to a note.  So in this PR I have done so.

You can setup a shortcut to append a specific content to any note, or you can get the note from another shortcut intent.  

The append intent uses the simperium api so the server can handle any diffing or sync that needs to happen and the main app does not need to be open for the shortcut to work and then sync.

relies on #1169 
### Test
1.  Build and Run Simplenote, just to make sure that you have saved the extension token so the intents can upload to simperium
2. run the intents extension on the shortcuts app
3. add a new shortcut, find Simplenote, select Append to note
4. add some content and pick a note to append to.  Run the shortcut
5. Launch simplenote, find that note, confirm the content was added to the note in question

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
> These changes do not require release notes.
